values/nginx-ingress/nginx-ingress.gotmpl

```diff
@@ -47,21 +47,17 @@ controller:
     v: 3
     enable-ssl-passthrough: true
   config:
-    ssl-redirect: {{ if $v.otomi.hasCloudLB }}"false"{{ else }}"true"{{ end }}
-    hsts: "true"
-    disable-ipv6: "true"
-    client-body-timeout: "5"
-    client-header-timeout: "5"
-    client-max-body-size: "0"
+    client-body-timeout: 5
+    client-header-timeout: 5
+    client-max-body-size: 0
+    disable-ipv6: true
```

Comment:
I have never tried it but does it mean we do not support IPv6 at all?

Reply:
too many of our components don't support it...maybe worth an investigation if that has changed, but I think the cloud providers don't support it (maybe google)